.github/workflows/benchmark_pull_request.yml

```diff
@@ -0,0 +1,67 @@
+name: benchmark_pull_request
+
+on:
+  push:
+    branches:
+      - 'hacl-star-**'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  default:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup
+        run: |
+          sudo apt-get update
+          sudo apt-get install ninja-build
+          sudo apt-get install python3 python3-scipy
+
+      - name: Checkout hacl-packages (${{ github.head_ref }})
+        uses: actions/checkout@v3
+        with:
+          path: pr
+
+      - name: Build hacl-packages (${{ github.head_ref }})
+        working-directory: pr
+        run: ./mach build --release --benchmarks
+
+      - name: Checkout hacl-packages (main)
+        uses: actions/checkout@v3
+        with:
+          path: main
+          ref: main
+
+      - name: Build hacl-packages (main)
+        working-directory: main
+        run: ./mach build --release --benchmarks
+
+      - name: Checkout google/benchmark
```

Comment:
Do we need this step? From my understanding we should already have a copy of gbenchmark at this state because cmake already fetched it, i.e., `./build/benchmark-src/tools/compare.py`?

Reply:
thanks for the remark! we now use the version fetched by cmake